querydsl-core/src/main/java/com/querydsl/core/support/SerializerBase.java

```diff
@@ -170,8 +168,12 @@ public void setAnonParamPrefix(String prefix) {
         this.anonParamPrefix = prefix;
     }
 
+    /**
+     * Not used anymore
```

Comment:
Or put that text in a `@deprecated` tag, since `// not used anymore` is already on the following line, once is enough.